Docs/ProjectSpec.md

```diff
@@ -373,7 +376,7 @@ targets:
 
 ### Build Script
 
-Run script build phases added via **prebuildScripts** or **postBuildScripts**. They run before or after any other build phases respectively and in the order defined. Each script can contain:
+Run script build phases added via **prebuildScripts**, **postCompileScripts**, or **postBuildScripts**. They run before any other build phases, after the Compile Sources build phase, or after any other build phases respectively and in the order defined. Each script can contain:
```

Comment:
Now that we have 3, maybe we need to split them into explicit sections:

Run script build phases can be added at 3 different points in the build:
- **prebuildScripts**: Before any other build phases
- **postCompileScripts** After the compile sources build phase
- **postbuildScripts**: After any other build phases

Each script can contain: